db/sql/0_msar.sql

```diff
@@ -2227,6 +2269,22 @@ BEGIN
       msar.process_col_alter_jsonb(tab_id, col_alters)
     );
   END IF;
+
+  -- Here, we perform all description-changing alterations.
+  FOR description_alter IN 
+    SELECT
+      (col_alter->>'attnum')::integer AS col_id,
+      quote_literal(col_alter->>'description') AS comment_
+    FROM jsonb_array_elements(col_alters) AS col_alter
+    WHERE __msar.jsonb_key_exists(col_alter, 'description')
+  LOOP
+    PERFORM __msar.comment_on_column(
+      tab_id := tab_id,
+      col_id := description_alter.col_id,
+      comment_ := description_alter.comment_
+    );
+  END LOOP;
```

Comment:
```suggestion
  -- Here, we perform all description-changing alterations.
  FOR description_alter IN 
    SELECT
      (col_alter->>'attnum')::integer AS col_id,
      col_alter->>'description' AS comment_
    FROM jsonb_array_elements(col_alters) AS col_alter
    WHERE __msar.jsonb_key_exists(col_alter, 'description')
  LOOP
    PERFORM msar.comment_on_column(
      tab_id := tab_id,
      col_id := description_alter.col_id,
      comment_ := description_alter.comment_
    );
  END LOOP;
```

This change lets you remove a function, simplifying the whole thing a bit.

Reply:
There might be conflict in parameter quoting. Let me see.